src/domUpdates.js

```diff
@@ -0,0 +1,252 @@
+import scripts from './scripts';
+import Pantry from './pantry';
```

Comment:
I am not sure why, but I am leery about having imports at the top of this file. Part of me feels like webpack or osmething else should be handling this. But I am glad it works one the less!